runbot/models/build.py

```diff
@@ -678,6 +678,14 @@ def _get_docker_name(self):
         self.ensure_one()
         return '%s_%s' % (self.dest, self.active_step.name)
 
+    def _get_error_tail_message(self, log_path):
+        lines = tail(log_path)
+        details = ''
+        if lines:
+            last_lines = '\n'.join(lines)
```

Comment:
readlines returns lines with a `\n`  at the end already.

```suggestion
            return '\n' + ''.join(lines)
```